views/partials/stock/product-offering/directives/priceplan-form.jade

```diff
@@ -77,4 +77,176 @@ form(name="form" novalidate)
         label
           span Enter a description
           small.text-muted (optional)
-        textarea.form-control(name="description" rows="2" ng-model="pricePlan.description")
\ No newline at end of file
+        textarea.form-control(name="description" rows="2" ng-model="pricePlan.description")
+  .row
+    .col-xs-12
+      .form-group
+        label Include
+        .item-radio
+          label.radio Price plan
```

Comment:
When you chose this option you are not creating a price plan inside a price plan, but a price plan with two price components, please rename the radio option